packages/animations/src/animation_metadata.ts

```diff
@@ -740,10 +740,10 @@ export function group(
  * @usageNotes
  * When you pass an array of steps to a
  * `transition()` call, the steps run sequentially by default.
- * Compare this to the `{@link animations/group group()}` call, which runs animation steps in
+ * Compare this to the `[group()](/api/animations/group)` call, which runs animation steps in
```

Comment:
```suggestion
 * Compare this to the [`group()`](/api/animations/group) call, which runs animation steps in
```

You can check the rendering here : https://ng-dev-previews-fw--pr-angular-angular-59162-adev-prev-23xkfz32.web.app/api/animations/sequence?tab=usage-notes 

The backticks should be inside the square brackets. The same applies to the other changes.